libr/util/file.c

```diff
@@ -1149,15 +1149,51 @@ R_API char *r_file_temp(const char *prefix) {
 	return res;
 }
 
+R_API char *r_file_temp_suf(R_NULLABLE const char *prefix, R_NULLABLE const char *suf) {
```

Comment:
Suf stands for suffix but in the filename jargon that would be extension. I think there can be a better name for this function

Reply:
I am pretty bad at naming things. How about `r_file_temp_ex`?